data.json

```diff
@@ -1,12 +1,12 @@
 {
   "data": [
     {
-      "name": "Rahul Bansal",
-      "batch": 2019,
-      "picture_url": "https://i.ibb.co/S0v6KWh/IMG-20201214-WA0000.jpg",
-      "twitter": "https://twitter.com/itz_rahul_rj",
-      "github": "https://github.com/rahulbansal0502",
-      "linkedin": "https://www.linkedin.com/in/rahulbansal52/",
```

Comment:
please do not modify/delete the existing code block instead add a new one with your own information.